be/src/http/action/stop_be_action.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
```

Comment:
Why K8S can not access the `stop_be.sh`?

Reply:
If want to stop cluster remotely, we need an agent program in the container to accept commands and then execute the `stop` script, but there is no agent in container.

Reply:
@chaoyli Since the smallest operating unit is pod in the k8s, we want to avoid pod reconstruction and only restart the container to achieve in-place update.

Reply:
I get it. 
There are two issues need to addressed.
1. the `k_starrocks_exit` is not useful to kill intime. Because there are many threads are waiting in background. You had to use exit().
2. The authentication is need to check. @Ielihs